tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,56 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file_no_unexpected_comma = """
+module UnexpectedComma (..) where
+var =
```

Comment:
using `var` as a variable name is unnecessary.  If the `=` doesnt explain that the lhs is a variable name, the reader should give up programming.

use either foo / bar / baz , or a / b / c, or some other similar pattern.